client/post-editor/style.scss

```diff
@@ -9,6 +9,18 @@
 	@include breakpoint( "<660px" ) {
 		padding: 0;
 	}
+
+	> .notice {
```

Comment:
I don't see `notice-list` used in this instance (maybe because it's a global notice?):

<img width="413" alt="screen shot 2015-12-18 at 2 25 17 pm" src="https://cloud.githubusercontent.com/assets/3011211/11908687/38eebdd2-a593-11e5-92d6-56cd6dc7fb30.png">